src/StarCircle.svg

```diff
@@ -1,4 +1,5 @@
-<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M9,2.06c3.83,0,6.94,3.11,6.94,6.94s-3.11,6.94-6.94,6.94-6.94-3.11-6.94-6.94,3.11-6.94,6.94-6.94M9,1.31C4.75,1.31,1.31,4.75,1.31,9s3.44,7.69,7.69,7.69,7.69-3.44,7.69-7.69S13.25,1.31,9,1.31h0Z" fill="black"/>
-<path d="M8.97,4.81c.16,0,.3.09.37.23l1.07,2.15,2.36.34c.16.02.29.13.34.28.05.15,0,.32-.11.43l-1.72,1.68.39,2.36c.03.16-.04.31-.17.41-.13.09-.3.1-.44.03l-2.1-1.11-2.1,1.11c-.14.07-.31.06-.44-.03-.13-.09-.19-.25-.17-.41l.39-2.36-1.72-1.68c-.11-.11-.15-.28-.11-.43.05-.15.18-.26.34-.28l2.36-.34,1.07-2.15c.07-.14.21-.23.37-.23ZM8.97,6.16l-.8,1.6c-.06.12-.18.21-.31.23l-1.74.25,1.27,1.24c.1.09.14.23.12.37l-.29,1.75,1.55-.82c.12-.06.27-.06.39,0l1.55.82-.29-1.75c-.02-.13.02-.27.12-.37l1.27-1.24-1.74-.25c-.13-.02-.25-.1-.31-.23l-.8-1.6Z" fill="black"/>
+<svg width="18" height="18" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
```

Comment:
I added `fill="currentColor"` to skip `removeAttrsIfSameFill` from removing the white background